adr/ADR-8.md

```diff
@@ -245,14 +247,18 @@ type KeyValueManager interface {
 
     // KeyValueBucketNames is used to retrieve a list of key value bucket
     // names.
-    KeyValueBuckerNames(ctx context.Context) KeyValueNamesLister
+    KeyValueBucketNames(ctx context.Context) KeyValueNamesLister
 
     // KeyValueBuckets is used to retrieve a list of key value bucket
     // statuses.
-    KeyValueBuckets(ctx context.Context) KeyValueLister
+    KeyValueBuckets(ctx context.Context) KeyValueStatusLister
 }
 ```
 
+`KeyValueBucketNames` provides a `KeyValueNamesLister`. `KeyValueBuckets` provides a `KeyValueStatusLister`. 
```

Comment:
I would mention here, that the KeyValue listers should have the same pattern used for returning the itemrs as the Stream listers, for consistency.

Reply:
I had this information in a separate paragraph below this block, so moved it inline.